src/development/data-and-backend/json.md

```diff
@@ -275,19 +275,21 @@ part '[[highlight]]user[[/highlight]].g.dart';
 /// JSON serialization logic to be generated.
 [[highlight]]@JsonSerializable()[[/highlight]]
 
-/// Every json_serializable class must have the serializer mixin.
-/// It makes the generated toJson() method to be usable for the class.
-/// The mixin's name follows the source class, in this case, User.
-class User extends Object with _$[[highlight]]User[[/highlight]]SerializerMixin {
+class User {
   User(this.name, this.email);
 
   String name;
   String email;
 
   /// A necessary factory constructor for creating a new User instance
-  /// from a map. Pass the map to the generated _$UserFromJson constructor.
+  /// from a map. Pass the map to the generated `_$UserFromJson` constructor.
   /// The constructor is named after the source class, in this case User.
   factory User.fromJson(Map<String, dynamic> json) => _$[[highlight]]User[[/highlight]]FromJson(json);
+
+  /// `toJson` is the convention for a class to declare support for JSON
+  /// serialization. The implementation simply calls the private, generated
+  /// helper method `_$UserToJson`.
+  Map<String, dynamic> toJson() => _$[[highlight]]User[[/highlight]]ToJson(this);
 }
```

Comment:
This is a super nit, but consider rewording the comment here from "support for JSON serialization" to "support for serialization to JSON." Because "JSON serialization" is often used as an umbrella term to mean both serialization and deserialization (and here you're talking specifically about the former on its own), it might be clearer for some beginners.

Reply:
done!